packages/mdc-snackbar/foundation.ts

```diff
@@ -88,12 +88,15 @@ export class MDCSnackbarFoundation extends MDCFoundation<MDCSnackbarAdapter> {
       this.adapter_.addClass(OPEN);
 
       this.animationTimer_ = setTimeout(() => {
+        const timeoutMs = this.getTimeoutMs();
         this.handleAnimationTimerEnd_();
         this.adapter_.notifyOpened();
-        this.autoDismissTimer_ = setTimeout(() => {
-          this.close(REASON_DISMISS);
-        }, this.getTimeoutMs());
-      }, numbers.SNACKBAR_ANIMATION_OPEN_TIME_MS);
+        if (timeoutMs !== -1) {
```

Comment:
Create a constant `INDETERMINATE` for this value.

Reply:
Added `INDETERMINATE` as a const to the numbers enum and replaced existing usages.